xchange-mexbt/src/main/java/com/xeiam/xchange/mexbt/MeXBTExchange.java

```diff
@@ -0,0 +1,67 @@
+package com.xeiam.xchange.mexbt;
+
+import si.mazi.rescu.SynchronizedValueFactory;
+
+import com.xeiam.xchange.BaseExchange;
+import com.xeiam.xchange.Exchange;
+import com.xeiam.xchange.ExchangeSpecification;
+import com.xeiam.xchange.mexbt.service.polling.MeXBTAccountService;
+import com.xeiam.xchange.mexbt.service.polling.MeXBTMarketDataService;
+import com.xeiam.xchange.mexbt.service.polling.MeXBTTradeService;
+import com.xeiam.xchange.mexbt.service.streaming.MeXBTExchangeStreamingConfiguration;
+import com.xeiam.xchange.mexbt.service.streaming.MeXBTStreamingService;
+import com.xeiam.xchange.service.streaming.ExchangeStreamingConfiguration;
+import com.xeiam.xchange.service.streaming.StreamingExchangeService;
+import com.xeiam.xchange.utils.nonce.CurrentTime1000NonceFactory;
```

Comment:
Change to use epoch time in ms is fine.

But be better to change the document as well: `apiNonce: An integer that must be increasing for each request (for example use the current unix timestamp)`